encrypt/encrypt.go

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package encrypt
+
+import (
+	"crypto/rand"
+	"fmt"
+
+	"github.com/GehirnInc/crypt"
+	// package requires import the hash method to blank
```

Comment:
Typo in comment. "import" ==> "importing"

Reply:
done.